app/gui/src/components/TabView/SelectableTab.vue

```diff
@@ -1,26 +1,49 @@
 <script setup lang="ts">
+import SvgIcon from '@/components/SvgIcon.vue'
+import TooltipTrigger from '@/components/TooltipTrigger.vue'
+import { Icon } from '@/util/iconMetadata/iconName'
 import { motion } from 'motion-v'
+import { computed } from 'vue'
+import CloseButton from '../CloseButton.vue'
 
 const selected = defineModel<boolean>('selected')
+const props = defineProps<{
+  icon?: Icon | undefined
+  label?: string | undefined
+  title?: string | undefined
```

Comment:
I feel `tooltip` will be less confusing than `title`, as the distinction with `label` is not clear from just naming.